src/components/flashcard/Flashcard.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import "./Flashcard.css";
+
+const FlashCard = ({ question, answer }) => {
+  const [isFlipped, setIsFlipped] = useState(false);
+  const [showHint, setShowHint] = useState(false);
+
+  var mockData = {
```

Comment:
In modern JavaScript, `let` and `const` are preferred over `var` for variable declarations. In the future we may allow users to update the values for a question/answer. So in this case we may want to use `let` instead of `const.` 

Here is an article that outline the difference between [var and let](https://www.programiz.com/javascript/let-vs-var)

Reply:
Totally agree. Thanks! 
Var data was indicated in the ticket, so I thought I needed to follow instructions as in the ticket.

Reply:
Updated as per your request.